File: 771.jewels-and-stones.go
package main

/*
 * @lc app=leetcode id=771 lang=golang
 *
 * [771] Jewels and Stones
 */

// @lc code=start
func numJewelsInStones(jewels string, stones string) int {
	result := 0
	for _, s := range stones {
		for _, j := range jewels {
			if s == j {
				result++
			}
		}
	}
	return result
}

// @lc code=end

File: 771.jewels-and-stones.rb
#
# @lc app=leetcode id=771 lang=ruby
#
# [771] Jewels and Stones
#

# @lc code=start
# @param {String} jewels
# @param {String} stones
# @return {Integer}
def num_jewels_in_stones(jewels, stones)
  result = 0
  stones.each_char do |stone|
    result += 1 if jewels.include?(stone)
  end
  result
end
# @lc code=end

